Frameworks/Ajax/Ajax/Sources/er/ajax/AjaxFileUploadRequestHandler.java

```diff
@@ -121,14 +121,9 @@ else if (formData.isFileUpload()) {
 							progress.dispose();
 							throw e;
 						}
-						FileOutputStream fos = new FileOutputStream(progress.tempFile());
-						try {
+						try (FileOutputStream fos = new FileOutputStream(progress.tempFile())) {
 							progress.copyAndTrack(uploadInputStream, fos, _maxUploadSize);
 						}
-						finally {
-							fos.flush();
```

Comment:
The flush method of FileOutputStream is empty so it does nothing. Additionally at that point the stream is not used anymore so even if there was something to flush it would have no influence.